### DISCUSSION TITLE
Anytone AT-578UVPRO Extension

### DISCUSSION THREAD

**Discussion by JebBaker** [hmatuschek/qdmr#263] [General]
QDMR Ver. 0.10.4
Radio AT-578UVPRO

Issue: Extension Tab showing incorrect extensions.

Radio is NOOTB and never been programmed. 
The extensions tab does not show the anytone extensions. It only shows tyt and radioddity extensions. 
![Screenshot from 2022-11-02 13-28-20](https://user-images.githubusercontent.com/83728055/199572313-d3e0549e-5708-417d-af26-c2539cc7df24.png)

**Comment by hmatuschek** [comment]
The AnyTone extensions are not yet complete, as these devices allow for a myriad of settings. Hence, only TyT and Radioddity extensions are shown.

**Comment by rhalley** [comment]
I have a D-578UVIIIPlus and I am glad to see this branch progressing again. - many thanks.  I am using FSQCall and I need to be able to change the microphone type from serial to voltage detect and back again for normal analog voice operation.  Do you have any plans for this specific configuration change, and if so an approximate time frame. Cheers Richard.

**Reply by hmatuschek** [reply]
Where do I find this setting? BTW, the D578UVIII (actually AT-D578UV2) is not supported yet. However, once I've updated the codeplug format for the AT-D578UV, I'll add the radio, as the codeplug is the same for both devices, the latter just identifies itself differently.

**Reply by hmatuschek** [reply]
Just as an information: I am working on the AnyTone settings extension (see issue #298), this will be part of the next release (0.11.3).

**Reply by rhalley** [reply]
The only place I have found it is in the Anytone windows app.  I go to:
Common Settings
  Optional Settings
    Other
      HandOpMode - either Level Detect or SerialPort Detect
Not sure if this is helpful. Cheers
